app/src/main/java/com/coveros/coverosmobileapp/MainActivity.java

```diff
@@ -3,13 +3,11 @@
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
-import java.util.function.Function;
-
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+        setContentView(R.layout.post_list);
```

Comment:
this will likely cause an issue when it comes time to merge, just as a future warning